lib/sidekiq/skylight/blacklisted.rb

```diff
@@ -0,0 +1,9 @@
+module Sidekiq
+  module Skylight
+    module Blacklisted
+      def blacklisted?
```

Comment:
Sorry, I thought I made a comment on this already. This name is too general here and could conflict with other methods in a job. I'd prefer something more specific to this gem, such as `blacklisted_for_skylight?`.